src/eterna/mode/GameMode.ts

```diff
@@ -20,8 +20,6 @@ import EternaURL from 'eterna/net/EternaURL';
 import Folder from 'eterna/folding/Folder';
 import Dialog from 'eterna/ui/Dialog';
 
-import NuPACK from 'eterna/folding/NuPACK';
-
```

Comment:
presumably this is back from when I was toggling things based on `this._folder.name === NUpACK.NAME` or something, so... good to kill this